modules/javafx.graphics/src/main/java/javafx/scene/image/WritableImage.java

```diff
@@ -75,6 +77,26 @@ public WritableImage(@NamedArg("width") int width, @NamedArg("height") int heigh
         super(width, height);
     }
 
+    private PixelBuffer<? extends Buffer> pixelBuffer = null;
+
+    /**
+     * Constructs a {@code WritableImage} using the specified {@code PixelBuffer}.
+     * The {@code java.nio.Buffer} provided by the {@code PixelBuffer} will be used
```

Comment:
I don't think you need the fully qualified name for Buffer since it's linked in the method arguments.